CONTRIBUTORS

```diff
@@ -5,8 +5,8 @@ Eileen Kuehn <eileen.kuehn@kit.edu>
 matthias.schnepf <matthias.schnepf@kit.edu>
 Rene Caspart <rene.caspart@cern.ch>
 Stefan Kroboth <stefan.kroboth@gmail.com>
-mschnepf <matthias.schnepf@kit.edu>
 R. Florian von Cube <florian.voncube@gmail.com>
+mschnepf <matthias.schnepf@kit.edu>
```

Comment:
Should these changes really be part of the commit? Is it possible to fix the order of contributors to not spam other PRs?

Reply:
The contributors file is managed by pre-commit, so currently this has to be part of the commit. Any ideas how to change the workflow without losing automatism?